CHANGELOG-3.6.md

```diff
@@ -0,0 +1,17 @@
+
+
+Previous change logs can be found at [CHANGELOG-3.5](https://github.com/etcd-io/etcd/blob/main/CHANGELOG-3.5.md).
+
+<hr>
+
+## v3.6.0 (TBD)
+
+See [code changes](https://github.com/etcd-io/etcd/compare/v3.5.0...v3.6.0) and [v3.5 upgrade guide](https://etcd.io/docs/latest/upgrades/upgrade_3_5/) for any breaking changes.
+
+### etcdctl v3
+
+- Add command to generate shell completion. ([PR#13133](https://github.com/etcd-io/etcd/pull/13133)).
```

Comment:
Nit: I believe this is more like in style to how the previous changelog was formatted, same with the other one: 

```suggestion
- Add command to generate [shell completion](https://github.com/etcd-io/etcd/pull/13133).
```

Reply:
Done